recipe/meta.yaml

```diff
@@ -14,26 +14,36 @@ source:
      - libca.patch
 
 build:
-  number: 1
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  number: 2
+  script: |
+      export NOLIBCA=1
+      echo "Using LIBCA from $PYEPICS_LIBCA"
+      rm -rf epics/clibs
+      "{{ PYTHON }} -m pip install . --no-deps -vv"
+
 
 requirements:
   host:
-      - python
-      - pip
+    - python
+    - pip
```

Comment:
Should we also use `epics-base` here as it's used in the `script` section?

Reply:
Does this code build any binaries?
If so then we need a build and host section.
Is epics-base needed to build the software (install the python code)?
Python should already be installed for testing so you don't need it in the test section I hope, since it is a runtime dep of this code.

Reply:
Just read you comment above, that is a nasty thing. I would have hoped that the python var would have propagated

Reply:
This does not build any binaries.

Reply:
Then you might not need `epics-base` in the host section? Unless that is needed by the `setup.py`

Reply:
I noticed that `      echo "Using LIBCA from $PYEPICS_LIBCA"` returned `Using LIBCA from ` string in CI, meaning `$PYEPICS_LIBCA` var wasn't defined.

Reply:
that is ok, it does not need to be anymore.